package.json

```diff
@@ -17,12 +17,9 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.30",
     "@fortawesome/free-brands-svg-icons": "^5.14.0",
     "@fortawesome/free-solid-svg-icons": "^5.14.0",
-    "@fortawesome/vue-fontawesome": "^0.1.10",
-    "aos": "^2.3.4",
     "core-js": "^3.6.5",
     "d3": "^5.16.0",
     "dns": "^0.2.2",
-    "flubber": "^0.4.2",
```

Comment:
Oh interesting, I didn't know we had flubber uploaded in here!

Reply:
I also had installed some other libraries to play around with and saw that they added themselves into the project, whoops. I think what I did should remove them entirely, but wiht keeping an eye on to make sure there isn't extra baggage slowing down site performance